plugin/trino-ldap-group-provider/src/main/java/io/trino/plugin/ldapgroup/LdapSingleQueryGroupProviderConfig.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.ldapgroup;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import jakarta.validation.constraints.NotEmpty;
+
+public class LdapSingleQueryGroupProviderConfig
+{
+    private String ldapUserMemberOfAttribute = "memberOf";
+
+    @NotEmpty
+    public String getLdapUserMemberOfAttribute()
+    {
+        return ldapUserMemberOfAttribute;
+    }
+
+    @Config("ldap.user-member-of-attribute")
```

Comment:
How about `ldap.user.member-of-attribute` - it is an attribute for a user right ?

Reply:
To all comments in this batch

The current config names are consistent with the LDAP authenticator https://trino.io/docs/current/security/ldap.html

Reply:
I think we might need to handle them there as well. `user` and `groups` are some subpart of it - But we could apply them by marking them as `legacy` annotations and we could migrate them.